api/src/main/java/uk/gov/hmcts/bar/api/data/enums/PaymentStatusEnum.java

```diff
@@ -5,8 +5,8 @@
 
 public enum PaymentStatusEnum {
 
-	DRAFT("D", "Draft"), PENDING("P", "Pending"), VALIDATED("V", "Validated"), PENDING_APPROVAL("PA",
-			"Pending Approval"), APPROVED("A", "Approved"), TRANSFERREDTOBAR("TTB",
+	DRAFT("D", "Draft"), PENDING("P", "Pending"), VALIDATED("V", "Validated"), PENDING_REVIEW("PA",
+			"Pending Approval"), REVIEWED("A", "Approved"), TRANSFERREDTOBAR("TTB",
```

Comment:
"Pending Approval" label should be renamed "Pending Review" and "Approved" -> "Reviewed" 
Renaming only the enum I don't think will make any visible changes

Reply:
This label is used in the front end and if you change that it breaks the front-end. Without outage we can't fix. Hence I didn't change the description.